addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -122,7 +122,7 @@ options.registry.SocialMedia = options.Class.extend({
                         anchorEl.classList.remove('d-none');// TODO remove in master.
                         this._removeSocialMediaClasses(anchorEl);
                     }
-                    const iEl = anchorEl.querySelector("i");
+                    const iEl = anchorEl.querySelector("i, span.fa");
```

Comment:
In 16.0 forward-port, probably adapt to use this: https://github.com/odoo/odoo/commit/4e33b5e4956da53558b0a36ae2e6da13220290ab#diff-d9bb92e950cfb526294e3c97c6c21879e5b2def8b7b70ddeb23960ef254d396bR1364